renovate.json

```diff
@@ -1,33 +1,57 @@
 {
 	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
 	"extends": [
-		"config:base"
+		"config:base",
+		":semanticCommits",
+		":gitSignOff"
 	],
 	"timezone": "Europe/Berlin",
 	"schedule": [
 		"every weekend"
 	],
-	"reviewers": ["datenangebot", "enjeck"],
-	"labels": ["dependencies"],
+	"labels": [
+		"dependencies",
+		"3. to review"
+	],
 	"rangeStrategy": "bump",
 	"rebaseWhen": "conflicted",
 	"ignoreUnstable": false,
 	"baseBranches": [
 		"main",
-		"stable0.7"
+		"stable0.8"
 	],
 	"enabledManagers": [
-		"npm"
+		"npm",
+		"composer",
+		"github-actions"
 	],
 	"ignoreDeps": [
 		"node",
 		"npm"
 	],
 	"packageRules": [
 		{
-			"matchUpdateTypes": ["major", "minor"],
-			"matchBaseBranches": ["stable0.7"],
-			"enabled": false
+            "description": "Request JavaScript reviews",
```

Comment:
Tabs instead of spaces?

Reply:
Ough, that was not caught by cs-fixer :crying_cat_face:  Good catch!

Committed the style fixes now!